integration_tests/utils.js

```diff
@@ -109,6 +110,25 @@ function replaceTestPath(string: string) {
   return string.replace(/^\W+(.*)integration_tests/gm, '<<REPLACED>>');
 }
 
+function fetchBundle(url: string): Promise<any> {
+  return new Promise((resolve, reject) => {
```

Comment:
Looks like we have a dependency on `node-fetch`. Maybe worth using it rather than wrapping `http.get` in a promise?

Reply:
I'm in the middle of fixing `start` test suite, which also replaces this with `fetch`